pkg/controller/installation/validation_test.go

```diff
@@ -1008,4 +1008,56 @@ var _ = Describe("Installation validation tests", func() {
 
 		})
 	})
+	Describe("validate CSIDaemonset", func() {
+		It("should return nil when it is empty", func() {
+			instance.Spec.CSINodeDriverDaemonSet = &operator.CSINodeDriverDaemonSet{}
+			err := validateCustomResource(instance)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("should return an error if it is invalid", func() {
+			instance.Spec.CSINodeDriverDaemonSet = &operator.CSINodeDriverDaemonSet{
+				Metadata: &operator.Metadata{
+					Labels: map[string]string{
+						"NoUppercaseOrSpecialCharsLike=Equals":    "b",
+						"WowNoUppercaseOrSpecialCharsLike=Equals": "b",
+					},
+					Annotations: map[string]string{
+						"AnnotNoUppercaseOrSpecialCharsLike=Equals": "bar",
+					},
+				},
+			}
+			err := validateCustomResource(instance)
+			Expect(err).To(HaveOccurred())
+
+			var invalidMinReadySeconds int32 = -1
+			instance.Spec.CSINodeDriverDaemonSet = &operator.CSINodeDriverDaemonSet{
```

Comment:
Looks like this version containing an invalid `MinReadySeconds` is never actually tested? Should it be removed or should there be some kind of `Expect` following this declaration?